fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/CascadesPlanner.java

```diff
@@ -318,6 +318,9 @@ public QueryPlanResult planQuery(@Nonnull final RecordQuery query, @Nonnull Para
                 .put(QueryPlanInfoKeys.TOTAL_TASK_COUNT, taskCount)
                 .put(QueryPlanInfoKeys.MAX_TASK_QUEUE_SIZE, maxQueueSize)
                 .put(QueryPlanInfoKeys.CONSTRAINTS, constraints)
+                .put(QueryPlanInfoKeys.STATS_MAPS,
+                        Debugger.getDebuggerMaybe().flatMap(Debugger::getStatsMaps)
```

Comment:
I take it that the query plan result won't be returned if the `STATS_MAP` is empty if the debugger isn't present? Maybe that's fine, as this information is a little internal. It does potentially limit the ability to get these stats through the relational server, but maybe we don't care about that (at least for the moment)

Reply:
That's right.